src/ert/_c_wrappers/enkf/_config_content_as_dict.py

```diff
@@ -165,7 +165,7 @@ def parse_signature_job(signature: str) -> Tuple[str, Optional[str]]:
     close_paren = signature.find(")")
     if close_paren == -1:
         raise ConfigValidationError(
-            errors=[f"Missing ) in FORWARD_MODEL job description {signature}"]
+            errors=[f"Missing ) in FORWARD_MODEL job description {signature!r}"]
```

Comment:
Would it be better to be explicit and use  `{repr(signature)}` instead of the more cryptic `!r`?

Reply:
It has been noted to be redundant: https://peps.python.org/pep-0498/#s-r-and-a-are-redundant . I guess we could change it to repr.